### ISSUE TITLE
Power connector in 1.2 is a bit precarious

### ISSUE THREAD

**Issue by reinauer** [arananet/A4000db#1]
I got the AmigaStore version of 1.2 and I died about 1000 deaths yesterday when I did a mainboard + daughterboard move and my machine would not turn on anymore. Turns out the power adapter is really close to the metal rail holding the daughterboard in place (and in my case, shorting it). See photos. I fixed it with kapton tape, but maybe it's possible to move the connector in future versions?

![IMG_20200717_103540](https://user-images.githubusercontent.com/10224538/87836357-a3396380-c844-11ea-81a8-b84f32268660.jpg)
![00000PORTRAIT_00000_BURST20200717103527415](https://user-images.githubusercontent.com/10224538/87836363-a6345400-c844-11ea-80fe-9cdf6c40bbc0.jpg)
![00100lrPORTRAIT_00100_BURST20200717103618064_COVER](https://user-images.githubusercontent.com/10224538/87836365-a6ccea80-c844-11ea-86da-9abc933c943f.jpg)
![00100lrPORTRAIT_00100_BURST20200717153206071_COVER](https://user-images.githubusercontent.com/10224538/87836367-a7658100-c844-11ea-97ce-92f98a679c7c.jpg)
![00100lrPORTRAIT_00100_BURST20200717153212151_COVER](https://user-images.githubusercontent.com/10224538/87836369-a7fe1780-c844-11ea-9774-c7b5cd17eb33.jpg)
![00100lrPORTRAIT_00100_BURST20200717153355717_COVER](https://user-images.githubusercontent.com/10224538/87836370-a896ae00-c844-11ea-9f76-f2974dd4362b.jpg)

**Comment by christopherpm** [comment]
Yeah, I have the same issue. If the power connector could be moved down, it would be ideal. The metal rail shorts against the power connector (both front and back)

**Comment by arananet** [comment]
oh! I see! will change it and upload the version! thanks for the update, for the current version you could solder 4 cables with a male connector to connect later to the PS.

Thanks for the feedback :)

**Comment by LinuxJedi** [comment]
Yep, I just had the 12v line on one go up in smoke due to not spotting this problem after installing it.

![PXL_20230810_154019696 MP](https://github.com/arananet/A4000db/assets/636271/b37defe1-361a-435f-94ed-5b0a79cf5231)

**Comment by arananet** [comment]
sorry to hear that, i've updated the readme.md regarding this, it's actually not required, and as I comment, should be isolated.

**Comment by Acill** [comment]
Yup, had thought I reported this at the beginning. I dont use this
connector and leave it off on the builds I do.
